File: Presentation/WebApp/ClientApp/src/Components/Core/Dto/AuditableDto.ts
export default interface IAuditableDto {
    uniqueId?: string;
    createdById?: string | null;
    createdOn?: Date | null;
    deletedById?: string | null;
    deletedOn?: Date | null;
    updatedById?: string | null;
    updatedOn?: Date | null;
}

File: src/Presentation/WebApp/Models/Dtos/AuditableDto.cs
namespace Temple.WebApp.Models.Dtos;
public class AuditableDto
{
    public long             Id              { get; set; }
    public long?            CreatedById     { get; set; }
    public DateTimeOffset?  CreatedOn       { get; set; }
    public long?            DeletedById     { get; set; }
    public DateTimeOffset?  DeletedOn       { get; set; }
    public long?            UpdatedById     { get; set; }
    public DateTimeOffset?  UpdatedOn       { get; set; }
    public Guid             UniqueId        { get; set; }
}
